main.go

```diff
@@ -187,6 +187,8 @@ func main() {
 			go SaveDataAsJson(data, username)
 		}
 
+		resultData.ResponseType = responseType
```

Comment:
Not super important, but can you default `responseType` to `default` if it is not one of `json` or `iframe`?